src/Components/Projects/Contribution/ContributionRow.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import ContributionBox from './Box';
+
+class ContributionRow extends React.Component {
```

Comment:
Change class component to functional component (preferably arrow function).

I know both are same in terms of performance, but functional components are more concise and modern (and are used everywhere else). They don't have any disadvantages but may get optimized in future. Please refer https://www.twilio.com/blog/react-choose-functional-components for more information.

Do let me know if you face any trouble with functional components.

Reply:
@tichnas but what about the life cycle method it not available in function component

Reply:
ok i will use useEffect there and it will be done

Reply:
@Bucky25 Are you saying about `componentDidMount`? If yes, there is a React hook called `useEffect` which you can use (refer to the link shared above)